bench/config/lets_encrypt.py

```diff
@@ -11,6 +11,7 @@
 from bench.config.production_setup import service
 from bench.config.site_config import get_domains, remove_domain, update_site_config
 from bench.utils import CommandFailedError, exec_cmd, update_common_site_config
+from distutils.spawn import find_executable
```

Comment:
Use `bench.utils.which` instead. We're trying to get rid of the distutils dependency.